docs/http_api.rst

```diff
@@ -113,10 +113,7 @@ type
     The string representing the type of permission.
 object_id
     A string that uniquely identifies the object that this permission acts
-    upon. "null" if this permission does not act on a specific object.
-metadata
-    A flat object that can be used to add any additional information that
-    might be needed for the permission.
+    upon. Empty string if this permission does not act on a specific object.
```

Comment:
If I'm reading that link correctly, it looks like they advise against allowing both `null` and empty strings, and mention that the django convention is to use empty strings. Not sure if I agree with the convention, `null` seems like a more explicit way of defining a permission not related to an object id, so that would be my vote, but I'm ok with us using empty strings.